### PR TITLE
Added remove from redis in invalidateAuthorization

### PR DESCRIPTION
**Pull request by munishchouhan**
This PR will fix remove the invlaid token from redis cache

### CODE REVIEW

File: src/main/groovy/io/seqera/wave/auth/RegistryAuthServiceImpl.groovy [hunk 1]
```diff
@@ -282,7 +282,7 @@ class RegistryAuthServiceImpl implements RegistryAuthService {
     void invalidateAuthorization(String image, RegistryAuth auth, RegistryCredentials creds) {
         final key = new CacheKey(image, auth, creds)
         cacheTokens.invalidate(key)
+        tokenStore.remove("key-" + key.stableKey())
```
Comment (pditommaso) [inline src/main/groovy/io/seqera/wave/auth/RegistryAuthServiceImpl.groovy:285]: Let's turn `"key-" + key.stableKey()` is a small private method to better track its usage

Comment (munishchouhan) [reply src/main/groovy/io/seqera/wave/auth/RegistryAuthServiceImpl.groovy]: done

**Comment by pditommaso** [comment]
Have you managed to test it?

**Comment by munishchouhan** [comment]
> Have you managed to test it?

yes, tested it in local

```
12:17:26.074 [io-executor-thread-5] DEBUG i.s.w.auth.RegistryAuthServiceImpl - Registry auth token for cachekey: 'RegistryAuthServiceImpl$CacheKey(image:biocontainers/biocontainers, auth:RegistryAuth(realm:https://auth.docker.io/token, service:registry.docker.io, type:Bearer, refreshable:true, endpoint:https://auth.docker.io/token?service=registry.docker.io), creds:RegistryCredentials[username=hrma017; password=dck****])' [key-05732d5fb2a5c830] => <TOKEN> [from store]
12:17:26.563 [io-executor-thread-4] DEBUG i.seqera.wave.http.HttpClientFactory - Creating new followRedirectsHttpClient: jdk.internal.net.http.HttpClientImpl@90ac776(2)
12:17:27.078 [io-executor-thread-4] DEBUG i.s.w.auth.RegistryAuthServiceImpl - Registry auth token for cachekey: 'RegistryAuthServiceImpl$CacheKey(image:biocontainers/biocontainers, auth:RegistryAuth(realm:https://auth.docker.io/token, service:registry.docker.io, type:Bearer, refreshable:true, endpoint:https://auth.docker.io/token?service=registry.docker.io), creds:RegistryCredentials[username=hrma017; password=dck****])' [key-05732d5fb2a5c830] => <TOKEN>
```

**Comment by munishchouhan** [comment]
Still getting 401 in dev.
I will dig more

**Comment by munishchouhan** [comment]
Ignor emy last comment, wrong version was deployed in dev
testing with correct version

**Comment by munishchouhan** [comment]
No more 401 error, tested successfully in dev

**Comment by munishchouhan** [comment]
```
docker pull wave.dev-tower.net/wt/xxxxxxx/biocontainers/biocontainers:v1.2.0_cv1
v1.2.0_cv1: Pulling from wt/xxxxxxx/biocontainers/biocontainers
a4a2a29f9ba4: Pull complete
127c9761dcba: Pull complete
d13bf203e905: Pull complete
4039240d2e0b: Pull complete
cd9d8a5907de: Pull complete
866d6c399416: Pull complete
edd3bf038310: Pull complete
926449ce537b: Pull complete
90bae14c26aa: Pull complete
eadce7eb4091: Pull complete
000c5021555d: Pull complete
52d2b413fa58: Pull complete
9ce3805c05ff: Pull complete
55217f90a8ca: Pull complete
Digest: sha256:3332df09060b4336e26929bb505a598d36137ad46eeb331dcb8846ca5776ca6c
Status: Downloaded newer image for wave.dev-tower.net/wt/xxxxxxx/biocontainers/biocontainers:v1.2.0_cv1
wave.dev-tower.net/wt/xxxxxxx/biocontainers/biocontainers:v1.2.0_cv1

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview wave.dev-tower.net/wt/xxxxxxx/biocontainers/biocontainers:v1.2.0_cv1
```

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.10.1
+1.10.2-A3
```
Comment (pditommaso) [inline VERSION:1]:
```suggestion
1.10.1
```

**Review by pditommaso — APPROVED** [review]

**Pull request merged**